ansible/roles/sysprep/tasks/main.yml

```
@@ -205,3 +206,11 @@
 - name: Disable Swap
   command: swapoff -a
   when: swapon.stdout
+
+- name: Disable firewalld if flag is set
+  systemd:
+    name: firewalld
+    enabled: false
+    masked: false
+    state: stopped
+  when: firewalld_disabled
```

Konvoy 1.x [documented](https://github.com/mesosphere/konvoy/blob/4ec8d8b32c72c6c8993d5316a23c61d4d5e94169/docs/site/include/os-svc-nodes.tmpl#L4) and [disabled](https://github.com/mesosphere/konvoy/blob/8d174ec0634d2d21728ea457e511de2adc57fa6c/ansible/roles/firewalld/tasks/main.yaml) firewalld. To me KIB not doing the same is bug.

Thoughts on disabling it by default but letting users keep it enabled by setting (`disabled_firewalld:true` in an override)?
```suggestion
  when: disable_firewalld
```

---

@some-things suggested we do it this way and leave it to be the user's responsibility to disable it.

---

I'm okay with enabled or disabled by default, as long as the user has the ability to toggle it.

---

I suggested to disable by default because thats what Konvoy1 was doing.
We know that having it enabled will prevent Kubernetes from working unless allow certain rules are added (we can't really know fully as these will depend on user k8s workloads too).

For more sophisticated users, the `disable_firewalld` can be used to enable after these rules have been applied.